css/site.styl

```diff
@@ -354,6 +354,9 @@ img
   img
     display: block
     width: 100%
+  &:hover 
+    .shop-product-title-holder, .shop-product-sold-text
```

Comment:
I think this hover should go all the way to the parent `article` `.shop-product` otherwise on every img hover on the site the browser is looking for 2 child classes and generally not finding them